docs/blog/2025-09-16-creating-an-nx-plugin-for-biome.md

```diff
@@ -0,0 +1,448 @@
+---
+title: 'Build Your Own Nx Plugin: Integrating Biome in 20 Minutes'
```

Comment:
I wouldn't necessarily lead with building an Nx plugin which might turn people off. Rather just mention using Biome with Nx or something like that.


Similarly in the intro paragraphs. Can we emphasize more that you don't **have to have** a plugin, but that simply will only make things smoother and your life easier if you have one?